docker-compose-3-persona.yml

```diff
@@ -118,6 +118,7 @@ services:
     command: /bin/sh -c "
       sleep 10 &&
       npx knex migrate:latest --knexfile build/lib/db/knexfile &&
+      sleep 10 &&
```

Comment:
This really doens't make sense to me. Why would we need a delay between db provisioning and flows provisioning? Do we just need to increase the first `sleep 10` to `sleep 20`?